scrapi/harvesters/bhl.py

```diff
@@ -0,0 +1,64 @@
+"""Harvests Biodiversity Heritage Library OAI Repository (BHL) metadata for ingestion 
+into the SHARE service.
+Example API call: http://www.biodiversitylibrary.org/oai?verb=ListIdentifiers&metadataPrefix=oai_dc&from=2015-02-01
+"""
+
+import re
+from scrapi.base import OAIHarvester
+from scrapi.base.schemas import OAISCHEMA
+from scrapi.base.helpers import updated_schema, default_name_parser
+
+
+def institution_name_parser(names):
+    ''' Parse institution names '''
+    contributor_list = []
+    for inst in names:
+        contributor = {
+            'name': inst,
+            'email': '',
+            'sameAs': []
+        }
+        contributor_list.append(contributor)
+
+    return contributor_list
+
+
+def aoi_process_contributors_bhl(*args):
+    ''' Parse people name for BHL'''
+    names = []
+    for arg in args:
+        if isinstance(arg, list):
+            for name in arg:
+                names.append(name)
+        elif arg:
+            names.append(arg)
+            
+    # Filter people names and clean dates and extra spaces        
+    people = [re.sub(r'\d+-(\d+)?', r'', n).strip() for n in filter(lambda x: ', ' in x, names)] or []
+    # Filter institution names and clean tabs
+    inst = [re.sub(r'\\t', r'', n).strip() for n in filter(lambda x: ', ' not in x, names) or []]
+        
+    if len(people)  > 0 and len(inst) > 0:
```

Comment:
E221 multiple spaces before operator
The first ```>``` operator has two spaces in front of it, python style guidelines recommend 1 space before and after each operator.